docs/reference/ml/apis/ml-api.asciidoc

```diff
@@ -115,6 +121,8 @@ include::get-snapshot.asciidoc[]
 include::get-calendar-event.asciidoc[]
 include::get-filter.asciidoc[]
 include::get-record.asciidoc[]
+//INFO
+include::ml-info.asciidoc[]
```

Comment:
If you commit the change to the page name, this include should be moved up to follow get-job-stats.